pkg/reconciler/events/pubsub/pubsub.go

```diff
@@ -47,6 +42,20 @@ const (
 	resourceGroup = "cloudpubsubsources.events.cloud.google.com"
 )
 
+// newReconciledNormal makes a new reconciler event with event type Normal, and
+// reason PubSubSourceReconciled.
+func newReconciledNormal(namespace, name string) pkgreconciler.Event {
+	return pkgreconciler.NewEvent(corev1.EventTypeNormal, "Updated", "Updated CloudPubSubSource: \"%s/%s\"", namespace, name)
```

Comment:
pkgreconciler.NewEvent(corev1.EventTypeNormal, "CloudPubSubSourceReconciled", "CloudPubSubSource reconciled: \"%s/%s\"", namespace, name)

Reply:
same for other sources